### PR TITLE
resolved "method and field have same name within a class"

### PR DESCRIPTION
**Pull request by maopeixia**
dev task : https://dev.azure.com/ceapex/Engineering/_workitems/edit/295101

### CODE REVIEW

File: src/Microsoft.Content.Build.MerfToSDPForJava.Interpreter/AbstractExpression.cs [hunk 1]
```diff
@@ -21,15 +21,26 @@ public virtual bool Interpret(PageModel pageModel, BuildContext context)
...
                 return;
             }
 
-            var fullFileName = fileName + Constants.Constants.YamlExtension;
-            string sdpymlfilePath = Path.Combine(StepUtility.GetSDPYmlOutputPath(_outputFolder), fullFileName);
+            string sdpymlfilePath = "";
+            string fullFileName = "", fullFileNameWithType = "";
+
+            fullFileName = fileName + Constants.Constants.YamlExtension;
+            fullFileNameWithType = (fileName + '-' + memberType ?? "") + Constants.Constants.YamlExtension;
```
Comment (anmeng10101) [inline src/Microsoft.Content.Build.MerfToSDPForJava.Interpreter/AbstractExpression.cs]: when will "memberType" be null?

File: src/Microsoft.Content.Build.MerfToSDPForJava.Interpreter/AbstractExpression.cs [hunk 2]
```diff
@@ -21,15 +21,26 @@ public virtual bool Interpret(PageModel pageModel, BuildContext context)
...
+
```
Comment (anmeng10101) [inline src/Microsoft.Content.Build.MerfToSDPForJava.Interpreter/AbstractExpression.cs:36]: memberType.ToLower()?

File: src/Microsoft.Content.Build.MerfToSDPForJava.Interpreter/AbstractExpression.cs (hunk 1 above)
Comment (maopeixia) [reply src/Microsoft.Content.Build.MerfToSDPForJava.Interpreter/AbstractExpression.cs]: resolved

File: src/Microsoft.Content.Build.MerfToSDPForJava.Interpreter/AbstractExpression.cs (hunk 2 above)
Comment (maopeixia) [reply src/Microsoft.Content.Build.MerfToSDPForJava.Interpreter/AbstractExpression.cs:36]: resolved

**Pull request merged**